docs/gettinginvolved/functional-testing.rst

```diff
@@ -134,7 +134,49 @@ To run the functional tests, run the following command:
 OpenStack
 =========
 
-The configuration stanza for an private OpenStack deployment is the same as Rackspace above, but ``auth_plugin`` should be included, which refers to an authentication plugin provided by ``python-keystoneclient``.
+The configuration stanza for a private OpenStack deployment is similar to Rackspace (above), with a few notable differences:
+
+* ``auth_plugin`` should be included, which refers to an authentication plugin provided by ``python-keystoneclient``, and
+* ``provider: "openstack"`` should be included, if the top level key is not ``openstack``.
 
 If required, you may need to add additional fields.
 For more information, see :ref:`openstack-dataset-backend`.
+
+Devstack
+--------
+
+It is assumed that you have a working ``Devstack`` environment.
+Refer to document "Setting up a devstack instance" on Google Drive.
```

Comment:
Yea.. I'm more okay with it being in our release-process docs than in functional testing docs. (Everyone knows only ClusterHQ releases flocker, but other people _might_ want to run functional tests if they are fixing a cinder bug or something.) Although, that would be planning for something that is otherwise unprecedented, so probably not worth it.

But, really I suppose it doesn't matter.